src/components/utils/WarningPanel.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { Alert } from 'antd'
+import { BareProps } from './types';
+
+export type WarningPanelProps = BareProps & {
+  desc: React.ReactNode,
+  action?: React.ReactNode
```

Comment:
Rename to `actions`. I'm planning to put there two buttons for Transfer Ownership: `Accept` & `Reject`